backend/server.ts

```diff
@@ -0,0 +1,62 @@
+import { executeQuery } from "./psql/queryExecute";
+import { QUERY_GET_ALL_DAOS } from "./psql/query";
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const { graphqlHTTP } = require("express-graphql");
+const {
+	GraphQLObjectType,
+	GraphQLSchema,
+	GraphQLString,
+	GraphQLInt,
+	GraphQLList,
+} = require("graphql");
+app.use(cors());
+
+const DaoType = new GraphQLObjectType({
+	name: "Dao",
+	description: "Dao Single Record",
+	fields: () => ({
+		app_id: { type: GraphQLInt, description: "app id" },
+		app_params: {
+			type: GraphQLString,
+			description: "app params like global state etc.",
+		},
+		asset_id: { type: GraphQLInt, description: "asset id" },
+	}),
+});
+
+const QueryRoot = new GraphQLObjectType({
```

Comment:
Move GraphQL define and config to new folder( how about move it to `graphql`).

Reply:
Done